packages/hash/api/src/model/ontology/util.ts

```diff
@@ -13,14 +13,14 @@ export const getNamespaceOfAccountOwner = async (
   params: { ownerId: string },
 ) => {
   const namespace =
-    params.ownerId === systemAccountId
+    params.ownerId === systemOrgAccountId
       ? SYSTEM_ACCOUNT_SHORTNAME
       : (
           (await UserModel.getUserById(graphApi, {
-            entityId: `${systemAccountId}%${params.ownerId}`,
+            entityId: `${systemOrgAccountId}%${params.ownerId}`,
```

Comment:
Can we use `entityIdFromOwnedByIdAndEntityUuid` for these please

Reply:
Yes, addressed in https://github.com/hashintel/hash/commit/6e6c358e80915bc8fb3fb132d66f3382fbfaf395